src/helpers/constants.js

```diff
@@ -0,0 +1,51 @@
+export const constants = {
+  // CAM API Swagger pages
+  swaggerPages: [
+    {
+      name: "account-mgmt",
+      url: "https://api.epa.gov/easey/dev/account-mgmt/swagger-json",
+      intro:
+        "Access account-based data for CAMD's allowance trading programs. This includes account information, account holdings, and transaction history for all account types. Allowance and emissions-based compliance records for facility accounts are also available.",
```

Comment:
I like the descriptions you have here! I need to replace the ones I made in the API docs page pull request with these